ui/ui-components/utils/appRefreshQueryHandler.ts

```diff
@@ -0,0 +1,4 @@
+import { Models } from "./apiData";
+import { EventDispatcher } from "./eventDispatcher";
+
+export class AppRefreshQueryHandler extends EventDispatcher<Models.AppType> {}
```

Comment:
I don't think this is used?  These `*Handlers` are for when more than one component on a page needs to /subscribe/ to changes in the UI.  The typical example is how a change on the `/account` page to the user's name should also update the `hello, {name}` in the navigation.  I think there's only one page/component at a time that would be displaying an AppDataRefresh